### PR TITLE
feat(vital-card): Add Article Card

### PR DESCRIPTION
**Pull request by VancheeZze**

### CODE REVIEW

File: packages/vital-card/src/components/ArticleCard/tokens/vitalArticleCard.ts [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+/**
+ * Default Article Card Token.
+ * By default Article Card has `Image`, `Title` and `Link` slots.
+ */
+const Default = asArticleCardToken({
+  ...Base,
+  Components: {
+    ...Base.Components,
+    Eyebrow: undefined,
```
Comment (hvanyo) [inline packages/vital-card/src/components/ArticleCard/tokens/vitalArticleCard.ts:53]: We probably want to make their wrappers undefined as well EyebrowWrapper & DescriptionWrapper

File: packages/vital-card/src/components/ArticleCard/tokens/vitalArticleCard.ts [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+ * Token that adds the Description slot to the Product Card.
+ * Adds the `DescriptionWrapper` slot as `P` element and the `Description` slot
+ * as default Plain Text Editor.
+ *
+ * Note: This token is meant to be layered on top of the `Product` token.
+ */
+const WithDescription = asArticleCardToken({
+  Components: {
+    DescriptionWrapper: replaceWith(P),
+    Description: on(EditorPlainClean)(vitalEditorPlain.Default),
```
Comment (hvanyo) [inline packages/vital-card/src/components/ArticleCard/tokens/vitalArticleCard.ts:110]: Can't remember were we going to keep editable stuff in components.  I know chris deleted ediors domain I think.

File: sites/vital-demo-next/src/data/pages/styleguide/article-card/description.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[
```
Comment (hvanyo) [inline sites/vital-demo-next/src/data/pages/styleguide/article-card/description.json:1]: can you move this page to /styleguide/card/article.  I like things filed nicely under their variations.

Comment (hvanyo) [reply sites/vital-demo-next/src/data/pages/styleguide/article-card/description.json:1]: can you also add article card to /styleguide page with link to this page.   NOTE you really need https://github.com/johnsonandjohnson/Bodiless-JS/pull/2193 to do this and then assign the category "vital 2.0" -- so do this in the feat/colors branch if it doesn't merge before yours.

File: packages/vital-card/src/components/ArticleCard/tokens/vitalArticleCard.ts (hunk 1 above)
Comment (VancheeZze) [reply packages/vital-card/src/components/ArticleCard/tokens/vitalArticleCard.ts:53]: `undefined` here really just removes the tokens applied to these slots in `Base`. Styles for `EyebrowWrapper` & `DescriptionWrapper` are not set in `Base` Components domain so adding `undefined` for these here would not make any difference. 

In this case these slots are coming from `cardComponentStart` and later adjusted for article card like this:
```
const articleCardComponentStart = {
  ...cardComponentStart,
  EyebrowWrapper: Fragment,
  Eyebrow: Fragment,
  Description: Fragment,
};
``` 

Here I set the `EyebrowWrapper` to Fragment and the `DescriptionWrapper` is the `Fragment` in `articleCardComponentStart`.

So basically both `EyebrowWrapper` & `DescriptionWrapper` are Fragments for the Article Card by default and only added in corresponding tokens.

File: packages/vital-card/src/components/ArticleCard/ArticleCardClean.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import { Fragment } from 'react';
```
Comment (dewen) [inline packages/vital-card/src/components/ArticleCard/ArticleCardClean.tsx:1]: License description for new files

File: packages/vital-card/src/components/ArticleCard/tokens/vitalArticleCard.ts [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+import {
+  vitalArticleCardElement, vitalTypography, vitalSpacing, DefaultDomains,
+} from '@bodiless/vital-elements';
+import {
+  flowHoc, extendMeta, TokenCollection, replaceWith, on, Div, P, as,
+} from '@bodiless/fclasses';
+
+import { CardComponents } from '../../Card/CardClean';
+import type { CardToken } from '../../Card/CardClean';
+import Base, { WithVerticalOrientation, WithHorizontalLeftOrientation, WithHorizontalContentCentered } from '../../Card/tokens/Base';
```
Comment (dewen) [inline packages/vital-card/src/components/ArticleCard/tokens/vitalArticleCard.ts:27]: break to 2 lines

File: packages/vital-card/src/index.ts [hunk 6]
```diff
@@ -15,3 +15,5 @@
 export * from './components/Card';
 export * from './components/FlowContainer';
 export * from './components/ProductCardTokens';
+
```
Comment (dewen) [inline packages/vital-card/src/index.ts:18]: extra line

**Review by dewen — DISMISSED** [review]

**Review by dewen — APPROVED** [review]

File: packages/vital-demo/src/styleguide/Page/StyleGuideTemplate/ArticleCard.tsx [hunk 7]
```diff
@@ -0,0 +1,130 @@
...
+  {
+    Horizontal: vitalArticleCard.WithHorizontalOrientation,
+    Vertical: vitalArticleCard.WithVerticalOrientation,
+  },
+);
+
+/*
+ * Compose the orientated cards with Descriptions & Eyebrows & Both toggles
+ */
+const DescriptonVariations = varyDesigns(
```
Comment (hvanyo) [inline packages/vital-demo/src/styleguide/Page/StyleGuideTemplate/ArticleCard.tsx:41]: typo

**Review by KaterinaPashkevich — APPROVED** [review]

**Review by KaterinaPashkevich — APPROVED** [review]
